_posts/2024-01-14-Institute of Asia and Africa MGU.md

```diff
@@ -0,0 +1,23 @@
+---
+title: انستیتو کشورهای آسیا و آفریقا وابسته به دانشگاه دولتی مسکو
+tags: [آموزش زبان فارسی حضوری, آموزش زبان فارسی الکترونیکی همزمان, تالیف کتاب, برگزاری آزمون تعیین سطح]
+date: 2024-01-14 12:30:00 +0330
+image: 
+    path: /assets/img/logos/Institute of Asia and Africa MGU.png
```

Comment:
It would be better not to use any space in image names to avoid possible URL problems.